File: 01-super-basic-questions/08-largest-ele-array/solution.py
n = int(input("Enter the size of the arry: "))

arr = []

# for _ in range(n):
#     arr.append(int(input("Enter element: ")))

arr = [int(input("Enter element: ")) for _ in range(n)]
    
max_ele = max(arr)
print(max_ele)

File: 01-super-basic-questions/08-largest-ele-array/solution.java
import java.util.Scanner;

class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter the size of the array: ");
        int num = sc.nextInt();

        int[] arr = new int[num];

        for (int i = 0; i < num; i++){
            arr[i] = sc.nextInt();
        }

        int maxEle = arr[0];

        for (int i = 0; i < num; i++){
            if (arr[i]> maxEle){
                maxEle = arr[i];
            }
        }

        System.out.println(maxEle);

    }
}